docs/source/contributing/branch-policy.md

```diff
@@ -33,14 +33,20 @@ legacy
     When opening a pull request, the contributor must open it against `main`.
     If the pull request is a feature or a bugfix, and if the release manager deems it useful to the latest version's branch, they may ask the contributor to backport it to that branch.
 
+`17.x.x`
+:   At the moment of this writing, `17.x.x` is the current stable branch in git.
+    Upon the final release of version `18.0.0`, the `17.x.x` branch line will become legacy.
+    This is the current actively developed branch in git, meaning that it may receive new features and bug fixes.
+    Any new feature will be released generally to the `main` branch, and only backported if the Volto Team approves it.
```

Comment:
I'd simplify a bit. I'm also concerned about the loose use of "legacy" throughout this policy. I think we need to explicitly define "legacy" as a term for Volto and all of Plone.

```
:   `17.x.x` is the current stable and actively developed branch in git.
    It may receive new features and bug fixes.
    Upon the final release of version `18.0.0`, the `17.x.x` branch line will become legacy.
    Any new feature will be generally released to the `main` branch, and only backported to older versions if the Volto Team approves it.
```

Reply:
It's the term we decided during the Volto Team Meeting at the conf. Don't know if we have to rethink it, but let's discuss it. I think @davisagli gave his ok during the meeting.
No, we decided that the "legacy" versions won't receive anything, unless it's justified and approved by the Team. 
This is to reduce the overall burden on the Release Manager (eg. when we have five legacy versions).

Reply:
jeez I just remembered you couldn't attend :( It would have been great to have you there, although few things, they were important. It's only now that I had time to write them down (and I'm cutting the 17.x.x branch now too).

Reply:
Why do we need YAOT (yet another obfuscating term)?

Let's just call it "unsupported" or "no longer supported" (which would be consistent with the term already used in the [Plone Release Schedule](https://plone.org/download/release-schedule)), and drop "legacy".

Reply:
+1 good enough for me

/cc @plone/volto-team @davisagli thoughts? If any, we will go with it.

Reply:
Updated suggestion, replacing "legacy" with "no longer supported".

Reply:
Done.